gallery/src/pages/AccessibilityChecklist.tsx

```diff
@@ -0,0 +1,1516 @@
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external href="https://www.w3.org/WAI/WCAG21/quickref/#audio-description-prerecorded">
+                Level AA
+              </NxTextLink>
+            </NxList.Actions>
+          </NxList.Item>
+          <NxList.Item className={`level-aaa ${isAAA && 'hidden'}`}>
+            <NxList.Text>
+              <NxStatefulCheckbox defaultChecked={false}>
+                Do we have a video recording of an ASL interpreter translating our pre recorded media?
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external href="https://www.w3.org/WAI/WCAG21/quickref/#sign-language-prerecorded">
+                Level AAA
+              </NxTextLink>
+            </NxList.Actions>
+          </NxList.Item>
+          <NxList.Item className={`level-aaa ${isAAA && 'hidden'}`}>
+            <NxList.Text>
+              <NxStatefulCheckbox defaultChecked={false}>
+                Have we provided an audio description of the prerecorded media that also includes implicit context that
+                may not be understood through a language translation alone?
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external
+                          href="https://www.w3.org/WAI/WCAG21/quickref/#extended-audio-description-prerecorded">
+                Level AAA
+              </NxTextLink>
+            </NxList.Actions>
+          </NxList.Item>
+          <NxList.Item className={`level-aaa ${isAAA && 'hidden'}`}>
+            <NxList.Text>
+              <NxStatefulCheckbox defaultChecked={false}>
+                Have we provided a transcript or equivalent version of live video content?
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external href="https://www.w3.org/WAI/WCAG21/quickref/#media-alternative-prerecorded">
+                Level AAA
+              </NxTextLink>
+            </NxList.Actions>
+          </NxList.Item>
+        </NxList>
+        <NxH4>
+          1.3 Adaptable- Create content that can be presented in different ways without losing the integrity andcontext
+          of the content.
+        </NxH4>
+        <NxList>
+          <NxList.Item className={`level-a ${isA && 'hidden'}`}>
+            <NxList.Text>
+              <NxStatefulCheckbox defaultChecked={false}>
+                Can we provide a simpler layout without losing information or structure?
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external
+                          href="https://www.w3.org/WAI/WCAG21/quickref/? showtechniques=111%2C128%2C133#adaptable">
```

Comment:
I suspect this part of the link isn't really supposed to be there
```suggestion
                          href="https://www.w3.org/WAI/WCAG21/quickref/#adaptable">
```